src/test/java/bio/terra/service/filedata/azure/tables/TableDirectoryDaoConnectedTest.java

```diff
@@ -256,6 +259,32 @@ public void testStorageTableMetadataDuringFileIngest() {
 
   }
 
+  @Test
+  public void testEnumerateFileRefEntries() {
+    String targetDirPath = String.format("/%s/", UUID.randomUUID());
+    List<FireStoreDirectoryEntry> fileList =
+        IntStream.range(0, 5)
+            .boxed()
+            .map(
+                i ->
+                    createStorageTableEntrySharedBasePath(
+                        targetDirPath, i + UUID.randomUUID().toString()))
+            .toList();
+
+    String collectionId = DATASET.toTableName(datasetId);
+    List<FireStoreDirectoryEntry> files =
+        tableDirectoryDao.enumerateFileRefEntries(tableServiceClient, collectionId, 0, 10);
+    assertThat(files, IsEqual.equalTo(fileList));
```

Comment:
This file already has a static import for `Matchers.equalTo`, which can be used here.
```suggestion
    assertThat(files, equalTo(fileList));
```